packages/api/db/cargo.sql

```diff
@@ -0,0 +1,36 @@
+CREATE SCHEMA IF NOT EXISTS cargo;
+
+-- Import dag cargo schema
+IMPORT FOREIGN SCHEMA cargo
+  LIMIT TO (aggregate_entries, aggregates, deals)
+  FROM SERVER dag_cargo_server
+  INTO cargo;
+
+-- Create materialized view from cargo "aggregate_entries" table
+CREATE MATERIALIZED VIEW public.aggregate_entry 
+    AS SELECT * FROM cargo.aggregate_entries;
+
+-- Indexes for "aggregate_entries" mat view
+CREATE UNIQUE INDEX aggregate_entry_unique_cidv1_aggregate_cid
+    ON public.aggregate_entry (cid_v1, aggregate_cid);
+CREATE INDEX aggregate_entry_cid_v1
+    ON public.aggregate_entry (cid_v1);
```

Comment:
Don't do this:
index (`A`, `B`)
index (`A`)

Instead do
index (`B`,`A`)
index (`A`)

This way searching for both `A` and `B` is properly indexed for you